src/structures/user.ts

```diff
@@ -37,6 +40,16 @@ export class User extends Base {
     return `<@${this.id}>`
   }
 
+  avatarURL(format: ImageFormats = 'webp', size: ImageSize = 512, dynamic: boolean = true): string {
+    return this.avatar != null
+      ? `${ImageURL(USER_AVATAR(this.id, dynamic ? this.avatar : this.avatar.replace('a_', '')), format, size)}`
```

Comment:
In my opinion it would be good to support format, size and dynamic options in an object.
Also, changing `ImageURL` function in structures/cdn.ts may help, as following:
```ts
/** Function to get Image URL from a resource on Discord CDN */
export const ImageURL = (
  url: string,
  format: ImageFormats = 'png',
  size: ImageSize = 128
): string => {
  if (url.includes('a_')) {
    return `${url}.${format === undefined ? 'gif' : format}?size=${size}`
  } else return `${url}.${format}?size=${size}`
}
```